### ISSUE TITLE
Nav bar not responsive in file browser

### ISSUE THREAD

**Issue by deadpahn** [asiekierka/nitrotracker#121]
So I can't find the other mention of this issue so this might be a duplicate ticket however I found a way to trigger this on my N3DSxl (dsi worked fine from what I recall).

I'm using the no wifi build on an r4 on the NEW 3DS XL. Just thought I would document that difference for trying to recreate this. I'm using a crappy microsd card, not sure if that is worth mentioning 

When I click away it can be functional again in the new folder.

**Comment by asiekierka** [comment]
What do you mean by "nav bar", exactly?

**Comment by deadpahn** [comment]
The scroll bar on the right side.
![image](https://github.com/user-attachments/assets/3f3b6155-a96e-488d-8943-895cc47d28d9)

**Comment by asiekierka** [comment]
Hmm. Is this just for directories which have fewer files than the list box's limit (no need to scroll)?

**Comment by deadpahn** [comment]
> Hmm. Is this just for directories which have fewer files than the list box's limit (no need to scroll)?

No, sorry for the confusion. That was just a random search engine image of NT to show where the scroll bar is located.

The issue usually happens when there are long drum loop files extending several pages beyond what's currently visible. Even after converting them to 8-bit, 32kHz mono, the problem persisted.

I’ll need to double-check, but I don’t recall encountering this issue when I had Nitrotracker installed on a DSi with its internal SD card.
